SSL improvements (#25106)
gradle/gradle: subprojects/docs/src/docs/release/notes.md

ADD RELEASE FEATURES BELOW
vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->

<a name="SSL"></a>
### SSL improvements

> Gradle had multiple issues when non-standard keystore and truststore were used.

Reviewer: ```suggestion
Gradle had multiple issues when non-standard keystores and truststores were used.
```

> This, for example, affected users on Linux systems with FIPS enabled and Windows users storing certificates in Trusted Root Certification Authorities store.

Reviewer: ```suggestion
This affected users on Linux systems with FIPS enabled and also Windows users who were storing certificates in the Trusted Root Certification Authorities store.
```
